pkg/util/kube/vm.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package kube
+
+const (
+	// CAPWClusterRoleLabelKey is the key for the label applied to a VM that was
+	// created by CAPW.
+	CAPWClusterRoleLabelKey = "capw.vmware.com/cluster.role" //nolint:gosec
+
+	// CAPVClusterRoleLabelKey is the key for the label applied to a VM that was
+	// created by CAPV.
+	CAPVClusterRoleLabelKey = "capv.vmware.com/cluster.role"
+)
+
+func HasTKGLabels(vmLabels map[string]string) bool {
```

Comment:
To be fair, this is really "HasCAPILabels", right?

Reply:
Yeah I just moved it from the bootstrap.go file which had this function name. Thought we should keep it as is, in case TKG VMs have other identified labels added in the future.